src/samples/EducationStart/index.tsx

```diff
@@ -361,6 +366,14 @@ const EducationStartCase: FunctionComponent<any> = () => {
     PCore.getStore().subscribe(() =>
       staySignedIn(setShowTimeoutModal, '', null, false, true, currentDisplay === 'resolutionpage')
     );
+
+    PCore?.getPubSubUtils().subscribe(
```

Comment:
Should we cover this section with a Unit test to ensure that the page is rendering as expected?
We could mock the isEduStartJourney, render the component, and verify if the correct data page is passed to the getPageData fucntion?